build.gradle.kts

```diff
@@ -36,9 +35,18 @@ plugins {
 
 buildscript {
     standardSpineSdkRepositories()
+
     dependencies {
-        @Suppress("RemoveRedundantQualifierName")
-        classpath(io.spine.internal.dependency.Spine.McJava.pluginLib)
+        classpath(variantOf(spine.mcJava) { classifier("all") })
+    }
+
+    configurations.all {
+        resolutionStrategy.force(
+            spine.protoData.plugin,
```

Comment:
`ProtoData` is a separate dependency object in recent versions of `config`. Why don't we migrate?

Reply:
This is not a `config` thing. `spine` here is a version catalog defined in `settings.gradle.kts`. Gradle imports it implicitly into `build.gradle.kts`.

I refrain from using `config` as much as possible for this repo, since `config` changes all the time and we need to substitute versions on the fly somehow. This is why the template mechanism for `settings.gradle.kts` was born.

However, we still need `config` for defining Maven repositories. Doing that by hand is hard and would require a lot of copying.